kfdef/kfctl_gcp_basic_auth.yaml

```diff
@@ -1,21 +1,10 @@
-# Please set project and email!
-apiVersion: kfdef.apps.kubeflow.org/v1alpha1
+apiVersion: kfdef.apps.kubeflow.org/v1beta1
 kind: KfDef
 metadata:
   creationTimestamp: null
   name: myapp2
```

Comment:
Here and every where else. Comment out name so that kfctl will auto provide a default value based on directory. Provide a comment explaining that people can uncomment it and explicitly set it to get the desired name

Reply:
done.